output_vt100.go

```diff
@@ -236,7 +236,7 @@ func (w *VT100Writer) SetColor(fg, bg Color, bold bool) {
 	if bold {
 		w.SetDisplayAttributes(fg, bg, DisplayBold)
 	} else {
-		w.SetDisplayAttributes(fg, bg, DisplayDefaultFont)
+		w.SetDisplayAttributes(fg, bg, DisplayDefaultFont, DisplayReset)
```

Comment:
it seems to be equivalent to following:

```
w.SetDisplayAttributes(fg, bg, DisplayReset)
```

Reply:
That was another thing I tried that seemed to work but I didn't want to take out the DisplayDefaultFont in case it did something I wasn't aware of.

Reply:
@c-bata any progress on merging it?